_pages/en/strategy-build-vs-buy-explained.md

```diff
@@ -17,406 +17,79 @@ permalink: /strategy-build-vs-buy-explained.html
+### Scope of This Document
 
-Time and again balling balloon on IT contracts have been observed.
-The Phoenix pay project started at $309M and is expected to now cost over $2.6B.
-The Email transformation initiative started at $x and was $100M by the time it was suspended.
-The contract for the content management system solution for the Canada.ca project started at $1.54M and was $14.9M by the end.
+The scope of this document is to provide you with different examples or scenarios which will help you to put the Adopt, Buy, Build strategy into context. It is not meant as a definitive guide. There are many different considerations and many different variables when it comes to decision making around whether to build or buy a solution or a component thereof. As such, we will be making a core set of assumptions that may not be applicable to your precise situation. Please refer to the actual strategy document for more detailed information.
 
-Though IT projects don't just end up over budget due to suicide bids.
-They can end up over budget due to a lack of strong integration with internal policy and subject matter experts on the project team.
-Without the input from these experts, projects end up not delivering on functionality.
-Fixing systems that don't meet functional requirements, comes at a cost, eats into the predicted cost savings for the new software solution.
-ETI was supposed to save $50M/year but was only completed up to 20% so any savings achieved are reduced by at least that amount.
-Coupled with the budget overruns, it would take 10+ years to achieve a return on investment now.
-Phoenix was expected to save $70M a year.
-Even if all those savings have been achieved, the increased cost means that the return on investment will take 37 years.
-The phoenix project is another example of this.
-It was launched with many important pay processing functions missing.
-Had the project been done in house, and/or with a tighter coupling with the subject matter experts, it is likely that a stronger case could have been made to not go forward without those functions.
+### Use Cases
 
-Further, offloading of risk to the taxpayer is not solely about the build decision itself but rather the ongoing operation.
-Often traditional build projects include a significant component of ongoing run and support.
-As such the ongoing support costs, and risk of being required to support companies solely due to severe vendor lock-in.
+Most of the time, we will not only adopt, buy, or build something.
+It's most often a combination of the 3.
+The following use cases show that some solutions can be classified in different locations of the diagram depending on how they are implemented and by who.
 
-### GC IT Project Analysis<!-- omit in toc -->
+![Basic Venn diagram with 3 overlapping circles for Adopt, Build and Buy. The middle is the intersection of all 3 (Adopt ^ Buy ^ Build)](assets/images/adopt-build-buy-venn.png)
```

Comment:
Since we say above that obtaining an IT solution is almost never just a build, just a buy or just an adopt, should those areas of the Venn diagram be smaller to reflect that?

Reply:
That's a good idea, let's see what we can do for the next release! :grinning: